### ISSUE TITLE
ERROR:  tids are not in order when building bitmap index 

### ISSUE THREAD

**Issue by cobolbaby** [greenplum-db/gpdb#14275]
### Greenplum version or build

- 6.19.1
- 6.23.0

### Actual behavior

```sql
CREATE INDEX idx_dw_fact_pca_yield_unit_model2
    ON dw.fact_pca_yield_unit USING bitmap
    (model)
    TABLESPACE tbs_ssd01;

ERROR:  tids are not in order when building bitmap index 18974716 (relfilenode 17044/17104/9291244): new tidnum 3846373408, last tidnum 3846373514 (bitmapinsert.c:1862)  (seg2 172.17.240.181:40000 pid=2434862) (bitmapinsert.c:1862)
SQL state: XX000
```

- seg2 log

```csv
2022-10-19 13:46:32.696854 CST,"gpadmin","TAO_BDC",p2434862,th116070528,"172.17.240.184","49764",2022-10-19 13:46:25 CST,56031738,con8797004,cmd10,seg2,,dx68564780,x56031738,sx1,"ERROR","XX000","tids are not in order when building bitmap index 18974716 (relfilenode 17044/17104/9291244): new tidnum 3846373408, last tidnum 3846373514 (bitmapinsert.c:1862)",,,,,,"CREATE INDEX idx_dw_fact_pca_yield_unit_model2
    ON dw.fact_pca_yield_unit USING bitmap
    (model)
    TABLESPACE tbs_ssd01;",0,,"bitmapinsert.c",1862,"Stack trace:
1    0xbf112c postgres errstart (elog.c:557)
2    0x762e1a postgres <symbol not found> (bitmapinsert.c:1856)
3    0x765711 postgres _bitmap_buildinsert (bitmapinsert.c:1792)
4    0x766d2e postgres <symbol not found> (bitmap.c:544)
5    0x792387 postgres <symbol not found> (index.c:2953)
6    0x794838 postgres IndexBuildScan (index.c:2550)
7    0x767838 postgres bmbuild (bitmap.c:93)
8    0xbfb68b postgres OidFunctionCall3Coll (fmgr.c:1669)
9    0x7931f9 postgres index_build (index.c:2363)
10   0x79405a postgres index_create (index.c:1256)
11   0x85a802 postgres DefineIndex (indexcmds.c:854)
12   0xa901d1 postgres <symbol not found> (utility.c:1548)
13   0xa8f045 postgres standard_ProcessUtility (utility.c:1049)
14   0x73f2ffb3e65a metrics_collector.so AdaptorProcessUtility + 0x72
15   0x73f2ffb3e99d metrics_collector.so EmitUtilityQueryInfo + 0x340
16   0xa8bc01 postgres <symbol not found> (palloc.h:158)
17   0xa8c6fe postgres <symbol not found> (pquery.c:1512)
18   0xa8de17 postgres PortalRun (pquery.c:1022)
19   0xa86319 postgres <symbol not found> (postgres.c:1389)
20   0xa8af7a postgres PostgresMain (postgres.c:5412)
21   0x6ae532 postgres <symbol not found> (postmaster.c:4489)
22   0xa0f982 postgres PostmasterMain (postmaster.c:1523)
23   0x6b24a1 postgres main (main.c:205)
24   0x73f30384e555 libc.so.6 __libc_start_main + 0xf5
25   0x6be1bc postgres <symbol not found> + 0x6be1bc
"
```

I don't know why the error is reported when creating a bitmap index for this table, but the others are right.

**Comment by SmartKeyerror** [comment]
Hi @cobolbaby, thanks for reporting this bug, but there lacks a step to reproduce the behavior, can you give us more detail about that?

**Comment by cobolbaby** [comment]
I don't know what information should be provided, can you give me some hints.

**Comment by SmartKeyerror** [comment]
The steps you have done to reproduce the behavior, include creating the table statement, the building index statement, etc. The whole information we can reproduce this bug in our environment, so we can debug and fix it.

**Comment by cobolbaby** [comment]
@SmartKeyerror The problem cannot be stably reproduced, only for this table, so it is difficult to construct test cases. I still hope you can tell me your potential doubts, and I will give feedback after investigation.

**Comment by SmartKeyerror** [comment]
Already been resolved by #15337, so close this issue. 

Please feel free to reopen it if you have any questions.
